rule_linux.go

```diff
@@ -82,7 +82,7 @@ func ruleHandle(rule *Rule, req *nl.NetlinkRequest) error {
 		req.AddData(rtAttrs[i])
 	}
 
-	native = nl.NativeEndian()
+	native := nl.NativeEndian()
```

Comment:
This change is part of the previous commit. Same PR.
You do not need an extra commit, just amend the previous one.